intranet/static/js/eighth/signupUI.js

```diff
@@ -174,25 +178,26 @@ function stickyHeaders(headers) {
 }
 
 $(function() {
-    var sh = new stickyHeaders($(".sticky-header"));
+    var sh = new StickyHeaders($(".sticky-header"));
     sh.load();
 
     $("#activity-list").on("scroll", function() {
         sh.scroll();
     });
 
     var initX = null, initY = null, listening = false;
-    $(".day-picker").on("touchstart", function(e) {
+    var $dayPicker = $(".day-picker");
+    $dayPicker.on("touchstart", function(e) {
         e.stopPropagation();
         initX = e.originalEvent.touches[0].clientX;
         initY = e.originalEvent.touches[0].clientY;
         listening = true;
     });
-    $(".day-picker").on("touchend", function(e) {
+    $dayPicker.on("touchend", function(e) {
         e.stopPropagation();
         listening = false;
     });
-    $(".day-picker").on("touchmove", function(e) {
+    $dayPicker.on("touchmove", function(e) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'anonymous' has a complexity of 5.](https://www.codacy.com/app/tjhsst-ion/ion/file/2687417334/issues/source?bid=3212528&fileBranchId=3299315#l200)